doc/getting_started/setup_fpga.md

```diff
@@ -58,19 +62,19 @@ While Bazel is the entry point for kicking off the FPGA synthesis, under the hoo
 During the build process, the boot ROM is baked into the bitstream.
 As mentioned above, we maintain two boot ROM programs, one for testing (_test ROM_), and one for production (_ROM_).
 
-To build an FPGA bitstream with the _test ROM_, use:
-```console
+To build an FPGA bitstream with the _test ROM_ for the CW340 board, use:
+```sh
 cd $REPO_TOP
-bazel build //hw/bitstream/vivado:fpga_cw310_test_rom
+bazel build //hw/bitstream/vivado:fpga_${BOARD}_test_rom
 ```
 
-To build an FPGA bitstream with the _ROM_, use:
-```console
+To build an FPGA bitstream with the _ROM_ for the CW340 board, use:
+```sh
 cd $REPO_TOP
-bazel build //hw/bitstream/vivado:fpga_cw310_rom
+bazel build //hw/bitstream/vivado:fpga_${BOARD}_rom
 ```
 
-Note, building these bitstreams will require Vivado be installed on your system, with access to the proper licenses, described [here](./install_vivado/README.md).
+**Note**: Building these bitstreams will require Vivado be installed on your system, with access to the proper licenses, described [here](./install_vivado/README.md).
 For general software development on the FPGA, Vivado must still be installed, but the Lab Edition is sufficient.
```

Comment:
```suggestion
> **Note**: Building these bitstreams will require Vivado be installed on your system, with access to the proper licenses, described [here](./install_vivado/README.md).
> For general software development on the FPGA, Vivado must still be installed, but the Lab Edition is sufficient.
```

Quoting this may be nice to indicate a different voice.

Reply:
Neat, done